components/emotionalRegulation/InterScreen.tsx

```diff
@@ -0,0 +1,58 @@
+import { View, Text, ScrollView, Pressable, StyleSheet } from 'react-native';
+import React from 'react';
+import { BLACK, PRIMARY, PURPLE, SIZES, TERTIARY } from '../../styles';
+import Title from '../Title';
+import { NavigationProp, useNavigation } from '@react-navigation/native';
+import { EmoRoutes } from './Navigation';
+
+export default function InterScreen() {
+  const { navigate } = useNavigation<NavigationProp<EmoRoutes>>();
+  return (
+    <>
+      <Title back color={PURPLE} text="Situationskontrolle" />
+      <ScrollView contentContainerStyle={styles.container}>
+        <Text style={[{ marginBottom: 30 }, styles.txt]}>Es ist Zeit deinen Tagesplan zu kontrollieren!</Text>
+        <View style={{ width: '100%' }}>
+          <Pressable
+            accessibilityHint="Beende die Übung"
+            onPress={() => navigate('NKontrolle')}
+            style={({ pressed }) => [{ backgroundColor: pressed ? PRIMARY : TERTIARY }, styles.btn]}>
+            <Text style={[{ marginVertical: 10 }, styles.txt]}>Alles klar!</Text>
+          </Pressable>
+          <Pressable
+            accessibilityHint="Beende die Übung"
+            onPress={() => navigate('Home')}
+            style={({ pressed }) => [{ backgroundColor: pressed ? PRIMARY : TERTIARY }, styles.btn]}>
+            <Text style={[{ marginVertical: 10 }, styles.txt]}>Heute lieber nicht</Text>
+          </Pressable>
```

Comment:
Wenn man beim kontrollieren des Tagesplans `Heute lieber nicht` auswählt wird die Bewertung des Starkmachers übersprungen. Denke die sollte auch beim Überspringen stattfinden oder?

Reply:
Können wir so machen, in Firma war nur kein weiterer Verlauf